File: sdk/src/wallet/bindings/nodejs/examples/destroy-alias.js
/**
 * This example will destroy an alias
 */
const getUnlockedManager = require('./account-manager');

async function run() {
    // Replace with an existing alias ID in your account
    const ALIAS_ID = '0x982667c59ade8ab8a99188f4de38c68b97fc2ca7ba28a1e9d8d683996247e152'

    try {
        const manager = await getUnlockedManager();

        const account = await manager.getAccount('0');

        await account.sync();

        let tx = await account.destroyAlias(ALIAS_ID);
        console.log(tx);
    } catch (error) {
        console.log('Error: ', error);
    }
    process.exit(0);
}

run();

File: sdk/src/wallet/bindings/python/examples/destroy_alias.py
from iota_wallet import IotaWallet

# In this example we will destroy an alias output

# TODO: replace with your own values.
ALIAS_ID = "0x08429fe5864378ce70699fc2d22bb144cb86a3c4833d136e3b95c5dadfd6ba0cef0500000000"
WALLET_PASSWORD = "some_hopefully_secure_password"

wallet = IotaWallet('./alice-database')

account = wallet.get_account('Alice')

# Sync account with the node
response = account.sync()

wallet.set_stronghold_password(WALLET_PASSWORD)

# Send transaction.
transaction = account.destroy_alias(ALIAS_ID)
print(transaction)
